app/controllers/admin/tutorials_controller.rb

```diff
@@ -3,10 +3,28 @@ def edit
     @tutorial = Tutorial.find(params[:id])
   end
 
-  def create; end
+  def import
+     @tutorial = Tutorial.new
+  end
+
+  def create
+    begin
+      tutorial = Tutorial.new(tutorial_params)
+      tutorial.import if tutorial.playlist_id
+    
+      tutorial.save
+              
+      flash[:success] = "Successfully created tutorial. #{view_context.link_to('View it here', tutorial_path(tutorial.id))}."
+      # flash[:success] = 'Successfully created Playlist. View it here' "i'm a typing person, i type things, look at me type. did you post it???? ?? ?? ?? "
+      rescue StandardError 
+        flash[:error] = 'Unable to create playlist.'  
```

Comment:
This is an amazing comment, but lets not forget to remove it before eval time. :D